### PR TITLE
Update installation documentation

### LINKED ISSUE #62: confirm installation process
Document how users can install a version of parastell that is outside the source tree and findable in the path, and confirm that imports work as expected.

### PR DESCRIPTION
**Pull request by connoramoreno**
Rewrites the installation instructions to reflect updates to dependencies. Also streamlines the installation process to reduce the number of steps. Confirmed to work with a new Conda test environment.

Closes #62.

### CODE REVIEW

**Comment by Edgar-21** [comment]
Should we add a yml environment file that matches these install instructions? Both for users and for use in the dockerfile for testing. I have one that is similar but not identical that I have been using while writing the dockerfile, but it would probably be best to have it match the install guide. 

We could also add a little paragraph mentioning that the conda and pip stuff can be obtained using the yml file.

**Comment by connoramoreno** [comment]
Oh, that's a good idea. I'll add a YML file soon.

File: README.md [hunk 1]
```diff
@@ -1,86 +1,91 @@
...
 
-```
-conda create -n parastell_env
-conda activate parastell_env
-conda install -c conda-forge mamba
-```
+Mamba is a faster, more reliable conda alternative that is fully compatible with conda packages. Mamba is available via conda (note that Miniforge ships with mamba already installed).
 
-The subsequent mamba and pip install commands should be run with this environment activated.
+### Option 1: Using the ParaStell Environment File
```
Comment (gonuke) [inline README.md:28]: What are your thoughts on only offering a preferred path using the `environment` on the README, and then referring people to another `INSTALL` doc for more details?  We could then streamline the info on the README to be a simple standard approach.

File: README.md [hunk 2]
```diff
@@ -1,86 +1,91 @@
...
+- [Miniforge](https://github.com/conda-forge/miniforge)
+- [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+- [Anaconda](https://www.anaconda.com/)
 
-```
-conda create -n parastell_env
-conda activate parastell_env
-conda install -c conda-forge mamba
-```
+Mamba is a faster, more reliable conda alternative that is fully compatible with conda packages. Mamba is available via conda (note that Miniforge ships with mamba already installed).
```
Comment (gonuke) [inline README.md:26]: Should we mention mamba only as part of option 2?

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for making these changes - just a few lingering suggestions and/or questions for discussion

File: README.md [hunk 3]
```diff
@@ -1,86 +1,91 @@
...
-pip install cad-to-dagmc
-pip install gmsh
+pip install netCDF4
 pip install pyyaml
 ```
-### Coreform Cubit
-Download and install version 2023.11 from [Coreform's Website](https://coreform.com/products/downloads/), then add the /Coreform-Cubit-2023.11/bin/ directory to the `PYTHONPATH` by adding a line to the .bashrc file like the following:
+
+## Install Coreform Cubit
+Download and install version 2023.11 from [Coreform's Website](https://coreform.com/products/downloads/), then add the `/Coreform-Cubit-2023.11/bin/` directory to your `PYTHONPATH` by adding a line similar to the following to your `.bashrc` file:
```
[the remark this answers is not in the document]
Comment (connoramoreno) [reply README.md:64]: For now, maybe we stick with 2023.11 until we can get the Svalinn plug-in issue straightened out with 2024.3?

File: README.md [hunk 4]
```diff
@@ -1,86 +1,91 @@
 ## ParaStell
-Parametric 3D CAD modeling tool for stellarator fusion devices. This toolset takes VMEC plasma equilibrium data to build intra-magnet components of varying thickness from a user-specified radial build, as well as corresponding coil filament point-locus data to build magnet coils of user-specified cross-section.
+Parametric 3-D CAD modeling toolset for stellarator fusion devices. This open-source Python package uses plasma equilibrium VMEC data and CadQuery to model in-vessel components of varying thickness in low-fidelity from a user-specified radial build. Furthermore, coil filament point-locus data corresponding to VMEC data and Coreform Cubit are used to model magnet coils of user-specified cross-section. Additional neutronics support includes the use of VMEC data and MOAB to generate tetrahedral neutron source definitions and Coreform Cubit to generate DAGMC geometries for use in Monte Carlo radiation transport software.
```
[the remark this answers is not in the document]
Comment (connoramoreno) [reply README.md:2]: Fair, I suppose that point doesn't need to be clarified

File: environment.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+name: parastell_env
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11.6
+  - mamba
```
[the remark this answers is not in the document]
Comment (connoramoreno) [reply environment.yml:6]: Yes it will, good catch. I suppose mamba isn't necessary here

File: README.md (hunk 1 above)
Comment (connoramoreno) [reply README.md:28]: That would be fine with me. Would the `INSTALL` document just be another Markdown file we refer users to? I'm not sure if an `INSTALL` document refers to some standard format.

File: README.md (hunk 2 above)
Comment (connoramoreno) [reply README.md:26]: Sure. The implementation of this option will depend on whether we include an `INSTALL` document

File: README.md (hunk 1 above)
Comment (gonuke) [reply README.md:28]: Just another markdown file, `INSTALL.md`.  You could start with this one and then whittle down the `README`

**Review by gonuke — APPROVED** [review]
Thanks @connoramoreno

**Pull request merged**